deploy/basic/terraform/provider.tf

```diff
@@ -1,11 +1,16 @@
-# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 # 
 
+terraform {
+  required_version = ">= 0.12.29"
+  # required_providers {
+  #   oci = ">= 3.90"
+  #   tls = ">= 2.0"
+  #   random = ">= 2.1"
+  # }
+}
+
 provider "oci" {
   tenancy_ocid = var.tenancy_ocid
-  # user_ocid        = "${var.user_ocid}"
```

Comment:
I know these are commented out, but the comments help people to run this locally without Resource manager. 
I think there is value in that.

Reply:
Yes. forgot to change. Will change to support both, ORM and local. Can completely remove as was before, as the main usage of mushop basic is ORM stack

Reply:
I've included because the ORM schema was not validating because do not had that entries.

Reply:
Fixed. Variables included to avoid ORM internal validation errors. Used default="" to be workable on ORM or plain terraform.